Header header
bool is_fault
FaultElement[] error_list
